### ISSUE TITLE
STOP CREATING ISSUES AND GOTO CANCANCAN INSTEAD

### ISSUE THREAD

**Issue by karlingen** [ryanb/cancan#1033]
https://github.com/CanCanCommunity/cancancan

**Comment by Senjai** [comment]
Sorry @karlingen  :( Wish there would be a better way to forward these short of creating a bot to do post automatically (which is probably against GH TOS)

**Comment by TurtleShip** [comment]
Thanks, @karlingen !

**Comment by joenas** [comment]
👍

**Comment by avocade** [comment]
Yes, very obviously discontinued. But please add a `DEPRECATED` section on top of the README then :D

https://github.com/CanCanCommunity/cancancan
